ngraph/python/tests/__init__.py

```diff
@@ -137,10 +137,12 @@ def xfail_test(reason="Mark the test as expected to fail", strict=True):
                                       "Argument element types are inconsistent.")
 xfail_issue_43742 = xfail_test(reason="RuntimeError: nGraph does not support the following ONNX operations:"
                                       "If")
-xfail_issue_43439 = xfail_test(reason="Check 'tensor_rank.is_static()' failed at "
-                                      "ngraph/core/src/validation_util.cpp:884:"
-                                      "map_1/while/select_bboxes/sort_bboxes_10/TopKV2 "
-                                      "Rank must be static in order to normalize negative axis=-1")
+xfail_issue_44602 = xfail_test(reason="While validating node 'v0::Squeeze"
```

Comment:
May be it make sense to find the root cause of 44602 and fix it in this PR also and remove the xfail?)

Reply:
I agree with Evgeny, it's better to fix the original problem. Looks like dynamic shapes appear somewhere in the network, it leads to issues in TopK op (fixed by relaxing shape inference of Loop op) and now in Squeeze op.

Reply:
Probably yes, but main intention was to avoid a large PR to review ;)

Reply:
^^ `Squeeze` case is rather separate problem, `map_1/while/select_bboxes/sort_bboxes_1/TopKV2:1` is fully static `{400}` after this change

Reply:
@lazarevevgeny @itikhono Problem with Squeeze exception was resolved. Unfortunately the model fails during execution because of dynamic shape Loop (we cannot determine number of iterations during shape inference because termination condition body output is not a constant). The problem should be resolved when CPU starts support dynamic shapes.